pkg/controller/importconfig/cluster_info.go

```diff
@@ -255,12 +298,11 @@ func getValidCertificatesFromURL(serverURL string, rootCAs *x509.CertPool) ([]*x
 	return retCerts, nil
 }
 
-func getBootstrapKubeConfigData(ctx context.Context, clientHolder *helpers.ClientHolder, cluster *clusterv1.ManagedCluster) ([]byte, []byte, error) {
-	importSecretName := fmt.Sprintf("%s-%s", cluster.Name, constants.ImportSecretNameSuffix)
-	importSecret, err := clientHolder.KubeClient.CoreV1().Secrets(cluster.Name).Get(ctx, importSecretName, metav1.GetOptions{})
+func getPreviousBootstrapKubeConfigData(ctx context.Context, clientHolder *helpers.ClientHolder, clusterName string) ([]byte, []byte, error) {
```

Comment:
wdym by getPreviousBootstrapKubeConfigData? is this a func that is supposed to be deprecated?